From: Russell Pagden <irra@rpagden.com>
Date: Mon, 11 Nov 2019 16:15:06 +0000
Subject: Skip shell detection tests

Skip tests that won't work in build chroot (shell detections)
---
 test/init.bats | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/init.bats b/test/init.bats
index ead63d8..6814699 100755
--- a/test/init.bats
+++ b/test/init.bats
@@ -25,12 +25,14 @@ load test_helper
 }
 
 @test "detect parent shell" {
+  skip
   SHELL=/bin/false run pyenv-init -
   assert_success
   assert_line "export PYENV_SHELL=bash"
 }
 
 @test "detect parent shell from script" {
+  skip
   mkdir -p "$PYENV_TEST_DIR"
   cd "$PYENV_TEST_DIR"
   cat > myscript.sh <<OUT
